### PR TITLE
Fix wrong usage of tracing in the sdk's module name trace package

### PR DESCRIPTION
**Pull request by bogdandrutu**
Signed-off-by: Bogdan Drutu <bogdandrutu@gmail.com>

### CODE REVIEW

File: sdk/tracing/build.gradle [hunk 1]
```diff
@@ -6,9 +6,9 @@ plugins {
     id "ru.vyarus.animalsniffer"
 }
 
-description = 'OpenTelemetry SDK Tracing'
-ext.moduleName = "io.opentelemetry.sdk.tracing"
-ext.propertiesDir = "build/generated/properties/io/opentelemetry/sdk/tracing"
+description = 'OpenTelemetry SDK Trace'
```
Comment (anuraaga) [inline sdk/tracing/build.gradle:9]: `Tracing SDK` seems correct

https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/sdk.md#tracing-sdk

I think a change of description from SDK Tracing to Tracing SDK is a good idea.

File: sdk/tracing/build.gradle [hunk 2]
```diff
@@ -6,9 +6,9 @@ plugins {
...
+ext.moduleName = "io.opentelemetry.sdk.trace"
```
Comment (anuraaga) [inline sdk/tracing/build.gradle:10]: These look correct for matching the package name, thanks

**Comment by anuraaga** [comment]
I would name the artifact `opentelemetry-tracing-sdk` to match the description / title of the spec page.

**Comment by anuraaga** [comment]
That being said, while `opentelemetry-tracing-sdk` sounds good to me, `opentelemetry-correlation-context-sdk` is strange. So `opentelemetry-sdk-tracing` seems fine too

File: sdk/tracing/build.gradle (hunk 1 above)
Comment (jkwatson) [reply sdk/tracing/build.gradle:9]: Yes, this should be "OpenTelemetry Tracing SDK"

Comment (bogdandrutu) [reply sdk/tracing/build.gradle]: Reverted. just update module name.

**Review by jkwatson — APPROVED** [review]

**Comment by jkwatson** [comment]
This file isn't used for anything programmatic at the moment, correct?

**Review by anuraaga — APPROVED** [review]

**Comment by anuraaga** [comment]
Thanks!

**Pull request merged**